modules/introduction/partials/dot-net-sdk-compat.adoc

```diff
@@ -0,0 +1,6 @@
+Use version 3.5.1 or later of the .NET SDK with Couchbase Server 7.6. 
+Earlier versions of this SDK have some compatibility issues.
+
+WARNING: v3.4.10 is incompatible with Server 7.6 and later--do not use. 
```

Comment:
Do we need to highlight 3.4.10?
It's been removed from NuGet, so people are unlikely to be able to install it.

Reply:
I can remove those if it's no longer an issue... I just added it because it was part of the SDK doc statement.